src/activity_actor.cpp

```diff
@@ -347,9 +347,10 @@ bool aim_activity_actor::load_RAS_weapon()
         return false;
     }
 
-    // Burn 0.2% max base stamina x the strength required to fire.
+    // Burn 0.2% max base stamina without cardio x the strength required to fire.
     you.mod_stamina( gun->get_min_str() * static_cast<int>( 0.002f *
-                     get_option<int>( "PLAYER_MAX_STAMINA" ) ) );
+                     ( get_option<int>( "PLAYER_MAX_STAMINA_BASE" ) +
+                       ( get_option<int>( "PLAYER_CARDIOFIT_STAMINA_SCALING" ) * you.base_bmr() / 2 ) ) ) );
```

Comment:
It seems curious to be doing a hardcoded stamina/cardio/BMR scaling with global game options inside a weapon-reloading activity. The concept of "max base stamina without cardio" might be better factored out into a helper function (especially if it's the kind of thing likely to be needed in other contexts).

Reply:
I am also unclear on why the character's weight (used by base_bmr()) would make a difference for weapon-reloading stamina use. Perhaps said helper function could assume a BMI of 25?

Reply:
It could simply be a flat number, or based on 3x the base stamina option.